### DISCUSSION TITLE
When will Mac Mouse Fix 2 come out of beta?

### DISCUSSION THREAD

**Discussion by ghost** [noah-nuebling/mac-mouse-fix#126] [Questions] [answered]
Do you have a timeline as to when you plan on releasing a stable version of Mac Mouse Fix 2? It looks promising!

**Comment by noah-nuebling** [comment, marked as answer]
Hi @JuNkY14257, 2.0.0 doesn't work for some people that could use 1.0.4 just fine, which is why I'm holding off on releasing it. But the issue is overall pretty rare and unlikely to affect you. Other than that Beta 11 is very stable. You can download it [here](https://github.com/noah-nuebling/mac-mouse-fix/releases/download/2.0.0-Beta-11/MacMouseFixApp.zip). Have fun :)
